diff --git a/src/Config/Repository.php b/src/Config/Repository.php
--- a/src/Config/Repository.php
+++ b/src/Config/Repository.php
@@ -194,7 +194,7 @@
         // If we've already loaded this collection, we will just bail out since we do
         // not want to load it again. Once items are loaded a first time they will
         // stay kept in memory within this class and not loaded from disk again.
-        if (isset($this->afterLoad[$namespace])) {
+        if ($namespace !== null && isset($this->afterLoad[$namespace])) {
             $items = $this->callAfterLoad($namespace, $group, $items);
         }
